src/util/Util.js

```diff
@@ -529,7 +529,11 @@ class Util extends null {
    * @deprecated Use {@link BaseMessageOptions#allowedMentions} instead.
    */
   static removeMentions(str) {
-    if (!deprecationEmittedForRemoveMentions) {
+    return Util._removeMentions(str, false);
+  }
+
+  static _removeMentions(str, fromCleanContent) {
+    if (!deprecationEmittedForRemoveMentions && !fromCleanContent) {
       process.emitWarning(
```

Comment:
This should be something like this:
```js
  static removeMentions(str) {
    if (!deprecationEmittedForRemoveMentions) {
      deprecationEmittedForRemoveMentions = true;
      process.emitWarning(
        // ...
      );
    }

    return Util._removeMentions(str);
  }

  static _removeMentions(str) {
    return str.replaceAll('@', '@\u200b');
  }
```

Likewise in `cleanContent`, remove the second parameter.

Reply:
> Also, does the lack of JSDoc automatically make this private/hidden from the docs?

Yep! By having no JSDoc, it's simply not parsed into the .json.